x-pack/plugins/ml/public/application/explorer/swimlane_container.tsx

```diff
@@ -286,6 +286,9 @@ export const SwimlaneContainer: FC<SwimlaneProps> = ({
     if (!showSwimlane) return {};
 
     const theme: PartialTheme = {
+      background: {
+        color: euiTheme.euiFormBackgroundColor,
```

Comment:
Is this the right color? I see a light shade of gray now rather than the page background color:

<img width="274" alt="image" src="https://user-images.githubusercontent.com/7405507/228512244-b9686871-7513-49c4-ac87-e08f78bc2aa3.png">

Reply:
I think you may need to use `euiPanelBackgroundColorModifiers.plain` because it seems to be the one used on your panel, but I'm not fully sure

Reply:
Thanks for the hint! Updated here [`48c59a3` (#153902)](https://github.com/elastic/kibana/pull/153902/commits/48c59a34f62e7697bedea2714de8d2a00ab01b80)

Here's now it looks now with regular and dark theme 
<img width="994" alt="Screen Shot 2023-03-29 at 11 35 23" src="https://user-images.githubusercontent.com/43350163/228607836-42f047e3-199d-4d2f-9538-b9cbc5a013cc.png">
<img width="994" alt="Screen Shot 2023-03-29 at 11 36 33" src="https://user-images.githubusercontent.com/43350163/228607843-87436ed4-7b6d-41fb-8100-9db3fa77e492.png">